src/olympia/tags/views.py

```diff
@@ -0,0 +1,24 @@
+from django.db.transaction import non_atomic_requests
+from django.utils.cache import patch_cache_control
+
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from .models import Tag
+
+
+class TagListView(APIView):
+    authentication_classes = []
+
+    def get(self, request, *args, **kwargs):
+        return Response(list(Tag.objects.values_list('tag_text', flat=True)))
+
+    @classmethod
+    def as_view(cls, **kwargs):
+        view = super().as_view(**kwargs)
+        return non_atomic_requests(view)
```

Comment:
Just curious, what does this do and why is it needed?

Reply:
we have [atomic](https://docs.djangoproject.com/en/3.2/topics/db/transactions/#django.db.transaction.atomic) requests enabled by default - that adds database queries at the start and end of the request cycle for transactions. This just disables that specifically for this view -because we know it won't ever result in database writes- for efficiency.